pkgs/development/tools/coursier/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  name    = "coursier-${version}";
+  version = "1.0.0-M15-5";
+
+  src = fetchurl {
+    url    = "https://github.com/coursier/coursier/raw/v${version}/coursier";
+    sha256 = "610c5fc08d0137c5270cefd14623120ab10cd81b9f48e43093893ac8d00484c9";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  phases = "installPhase";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ${src} $out/bin/coursier
+    chmod +x $out/bin/coursier
+    wrapProgram $out/bin/coursier --prefix PATH ":" ${jre}/bin ;
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
Would you like to add yourself as a maintainer?

Either way, can you add me, as I will be using this?